src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+
+.body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
```

Comment:
You have applied the margin to the right side of all stars except the last one. It's a good practice to be consistent with the margins. However, in your case, if you need to add more stars or elements to the left side of the stars, you would also need to manage their margins. To avoid this, you can apply the margin to the left side of all stars except the first one. It's just a suggestion for improvement, the current code works fine.